demos/JWST/S3_nirspec_fs_template.ecf

```diff
@@ -27,6 +27,7 @@ bg_thresh       [4,4]       # Double-iteration X-sigma threshold for outlier rej
 # Background parameters
 bg_hw           7           # Half-width of exclusion region for BG subtraction (relative to source position)
 bg_deg          0           # Polynomial order for column-by-column background subtraction, -1 for median of entire frame
+bg_method		med		    #Options: std (Standard Deviation), med (Median Absolute Deviation), mean (Mean Absolute Deviation)
```

Comment:
Watch the formatting in the template files.  You should be using soft tabs (i.e., spaces) instead of actual tabs.  Indents should be 4 spaces.

You'll also want to update the MIRI template files for S1 and S3 and the NIRCam template file for S3.